### DISCUSSION TITLE
Dragon NaturallySpeaking together with SEB

### DISCUSSION THREAD

**Discussion by yvonnekeller** [SafeExamBrowser/seb-mac#92] [Q&A] [answered]
Hello 
I have a request from a student who is visually impaired and uses Dragon NaturallySpeaking software for speech recognition. She uses the software via her ipad, then submits the converted text via "AirDrop" to her Mac, where she consolidates the text to Word. She then submits the Word file as an exam. What would be the solution here with the SEB? Installing the SEB on ipad & Mac? Is the "AirDrop" function still possible then? I have also heard that using the Dragon software together with the SEB is difficult? Can anyone give me advice on how the SEB would make sense here? 

<img width="314" alt="Dragon_SEB" src="https://user-images.githubusercontent.com/84328479/118498853-b815aa80-b726-11eb-9c1a-b0426cad1bf0.PNG">

**Comment by danschlet** [comment, marked as answer]
We will have a closer look into accessibility features for SEB macOS/iOS later this year. Currently I cannot answer your question, as we didn't try out tools like Dragon NaturallySpeaking. As you might have seen, there is an option 'Allow to use dictation ' in Preferences/[Security pane](https://safeexambrowser.org/macosx/mac_usermanual_en.html#AppearancePane). So you could try to use the macOS built-in dictation feature instead of a third party tool together with SEB. The classic kiosk lockdown mode of SEB doesn't allow any third party tools to display anything on the screen during an exam, and allowing applications to run during an  exam wasn't possible at all. In the upcoming macOS 12 Monterey, Apple added an option to the macOS built-in kiosk lockdown mode (called AAC/Assessment Mode) to allow additional applications to run alongside the secure assessment app and we will support that in SEB. That might help in using apps like Dragon NaturallySpeaking in exams.

If AirDrop would be used securely with SEB we also have to investigate later this year. We first have to work on some essential updates.

**Reply by yvonnekeller** [reply]
Thanks very much for your answer. this sounds very interesting and we are looking forward to the new features.
